### PR TITLE
Updated to react & react-dom v15.0.x. Updated lab to v10.x.x

### PR DESCRIPTION
**Pull request by byoigres**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
     "hoek": "3.x.x"
   },
   "peerDependencies": {
-    "react": "0.14.x ",
+    "react": "15.0.x",
     "react-dom": "0.14.x "
```
Comment (jedireza) [inline package.json]: I think the build is failing because `react-dom` in `peerDependencies` should also be `15.x.x`

**Comment by jedireza** [comment]
Also instead of `15.0.x` let's use `15.x.x` everywhere.

BTW, thanks for creating a PR!

**Comment by byoigres** [comment]
Opps, I didn't notice it. Going to fixed it.

**Comment by jedireza** [comment]
Thanks!

**Pull request merged**